src/app/components/mno-apps/mno-app-listing.html

```diff
@@ -31,9 +31,26 @@
         </div>
       </div>
     </div>
-
+    <!-- Public Marketplace Nav -->
+      <div class="col-sm-6 col-md-6 col-lg-4 landing-page-search-wrapper" ng-if="$ctrl.publicPage">
```

Comment:
For readability, I think we should use `ng-show="$ctrl.publicPage"` here instead of `ng-if`. (`ng-show` and `ng-hide` both apply styling to hide or show elements, while `ng-if` actually adds/removes DOM elements.)